.ado/templates/win32-nuget-publish.yml

```diff
@@ -0,0 +1,27 @@
+steps:
+  - checkout: self
+    persistCredentials: true
+
+  - template: setup-repo-min-build.yml
+
+  - script: |
+      yarn bundle
+    workingDirectory: apps/win32
+    displayName: 'Bundling win32'
+
+  # Pack the NuGet package
+  - task: CmdLine@1
+    displayName: 'Create NuGet package for Win32 FluentTester bundle.'
+    inputs:
+        filename: nuget
+        arguments: 'pack Microsoft.FluentUI.FluentTesterWin32.nuspec -OutputDirectory $(Build.ArtifactStagingDirectory)/nuget -OutputFileNamesWithoutVersion -Verbosity detailed -Version $(Build.BuildNumber)'
+        workingFolder: 'apps/win32/nuget'
```

Comment:
Do you have to create this folder beforehand or anything? The apple NuGet makes a `DerivedData` intermediate folder off of root, and I think that works fine without creating the folder beforehand.